### PR TITLE
Add multi-node-specific settings and other tweaks

### PR DESCRIPTION
**Pull request by erimatnor**
Information on configuration parameters for multi-node has been
added. This will help users get the best possible query performance
from their systems.

Some tweaks to existing sections are also included.

### CODE REVIEW

File: getting-started/setup-multi-node-basic.md [hunk 1]
```diff
@@ -52,17 +52,9 @@ You can now create distributed hypertables using
 use distributed hypertables as a non-superuser, the user role needs to
 exist on all data nodes and have the correct privileges to use the
 data nodes on the access node. Please refer to the next section for
-instruction on how to create roles on all data nodes and grant data
+instructions on how to create roles on all data nodes and grant data
 node privileges.
 
-It should now be possible to create distributed hypertables, e.g.,
```
Comment (erimatnor) [inline getting-started/setup-multi-node-basic.md:58]: Forgot to remove this in earlier PR. It's the same information as paragraph above.

**Review by pmwkaa — APPROVED** [review]

File: getting-started/setup-multi-node-auth.md [hunk 2]
```diff
@@ -11,11 +11,23 @@ To start, you'll need to have the following:
 
 ## Multi-node configuration
 
-In additions to the [normal configuration for TimescaleDB][configuration], it is also necessary to
-change the parameter `max_prepared_transactions` to a non-zero value on all nodes
-(if not already set, '150' is recommended). The parameter is located in
-`postgresql.conf`, typically in the data directory. If it isn't there, connect to
-the node (`psql`) and get the path with:
+In additions to the [normal configuration for
```
Comment (solugebefola) [inline getting-started/setup-multi-node-auth.md:14]:
```suggestion
In addition to the [normal configuration for
```

File: using-timescaledb/distributed-hypertables.md [hunk 3]
```diff
@@ -259,6 +259,22 @@ pushing down the `now()` function, the function is constified on the
...
+### Settings for improving query performance
+
+There are a number of configuration parameters that are important for
+good query performance on a distributed hypertable. These following
+settings apply to the access node.
+
+* `enable_partitionwise_aggregate` should be set to `on`.
+* `jit` should be set to `off` as JIT compilation currently doesn't
+  work well with distributed hypertables.
+* `timescaledb.remote_data_fetcher` should be set to `cursor` (the
```
Comment (solugebefola) [inline using-timescaledb/distributed-hypertables.md:271]: Not clear what the directive is here. Should the user set `timescaledb.remote_data_fetcher` to `cursor` or `rowbyrow`?

**Review by solugebefola — APPROVED** [review]

**Review by mfreed — APPROVED** [review]

File: using-timescaledb/distributed-hypertables.md (hunk 3 above)
Comment (erimatnor) [reply using-timescaledb/distributed-hypertables.md:271]: Yeah, it's up to the user. There are caveats with `rowbyrow`, so not recommended by default. Not sure how

**Pull request merged**